user/showforum.php

```diff
@@ -354,4 +375,4 @@ function threads($key)
 print generate_page($forum['name'], $tpl->parse("content", "showforum"), false, $meta_robots);
 
 // vim: sw=2
-?>
+?>
```

Comment:
No EOF on EOL, fix your editor :)

Reply:
Fixed issues.

Reply:
Still seeing no EOL on EOF ?

Reply:
also please rebase/squash thrashing commits :)

Reply:
Hopefully I sucked less this time.